igdiscover/init.py

```diff
@@ -317,11 +319,16 @@ def create_symlink(readspath, dirname, target):
 			sys.exit(1)
 		create_symlink(reads1, args.directory, target)
 
-	# Write the configuration file
 	configuration = pkg_resources.resource_string('igdiscover', Config.DEFAULT_PATH).decode()
-	with open(os.path.join(args.directory, Config.DEFAULT_PATH), 'w') as f:
+	# Write the configuration file
+	local_config_path = os.path.join(args.directory, Config.DEFAULT_PATH)
+	with open(local_config_path, 'w') as f:
 		f.write(configuration)
 
+	# Update defaults that are human specific
+	if args.species.lower() == 'human':
+		Config.update_option(local_config_path, [('preprocessing_filter.v_coverage', '97')])
```

Comment:
I’m not asking to change this now, but we should find a mechanism to have these settings outside of the code. There are alreday three "default" values for `v_coverage` in different places: one in `filter.py`, one in the `igdiscover.yaml` template and one in `config.py` (more if you count both germline and pre-germline filtering settings). This adds a fourth.